magstock/templates/regextra.html

```diff
@@ -28,7 +30,9 @@
         }
     };
     $(function () {
```

Comment:
Another unrelated fix; we should only move the food checkbox around if it actually exists :)